Makefile.cache

```diff
@@ -295,13 +295,17 @@ define SAVE_INTO_CACHE
 		echo "Target $(1) dependencies are modifed - global save cache skipped" >> $($(1)_DST_PATH)/$(1).log
 		$(eval $(1)_CACHE_DIR := $(SONIC_DPKG_LOCAL_CACHE_DIR))
 	 )
-	$($(1)_CACHE_USER) tar -C $($(1)_BASE_PATH) -mczvf $($(1)_CACHE_DIR)/$(MOD_CACHE_FILE) $(2) $(addprefix $($(1)_DST_PATH)/,$($(1)_DERIVED_DEBS) $($(1)_EXTRA_DEBS) )  \
+	$($(1)_CACHE_USER) tar -C $($(1)_BASE_PATH) -mcvf $($(1)_CACHE_DIR)/$(MOD_CACHE_FILE).tar $(2) $(addprefix $($(1)_DST_PATH)/,$($(1)_DERIVED_DEBS) $($(1)_EXTRA_DEBS) )  \
 		1>>$($(1)_DST_PATH)/$(1).log
-	sudo chmod 777 $($(1)_CACHE_DIR)/$(MOD_CACHE_FILE)
 
 	echo "File $($(1)_CACHE_DIR)/$(MOD_CACHE_FILE) saved in cache " >> $($(1)_DST_PATH)/$(1).log
 	echo "[ CACHE::SAVED ] $($(1)_CACHE_DIR)/$(MOD_CACHE_FILE)" >> $($(1)_DST_PATH)/$(1).log
 
+	$($(1)_CACHE_USER) tar -C $($(1)_BASE_PATH) -muvf $($(1)_CACHE_DIR)/$(MOD_CACHE_FILE).tar $($(1)_DST_PATH)/$(1).log
```

Comment:
The log file in the cache should have a different name with the current log file.
When building a target, it will add some log in the <target>.log. Will the log be overridden by the cache one?
Current log:
target/debs/bookworm/libnl-3-200_3.5.0-1_amd64.deb.log
Cache log:
target/debs/bookworm/libnl-3-200_3.5.0-1_amd64.deb.log.cache  -- You can give a better name.

Reply:
Handled. the cached log has .cache.log prefix